task_processing/plugins/kubernetes/task_config.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from pyrsistent import field
+from pyrsistent import PMap
+from pyrsistent import PVector
+
+from task_processing.interfaces.task_executor import DefaultTaskConfigInterface
+
+
+class KubernetesTaskConfig(DefaultTaskConfigInterface):
+    node_selector = field(type=PMap)
+    containers = field(type=PVector)
+    # Hardcoded for the time being
+    restart_policy = "Never"
+    # By default, the retrying executor retries 3 times. This task option
+    # overrides the executor setting.
+    retries = field(type=int,
+                    factory=int,
+                    mandatory=False,
+                    invariant=lambda r: (r >= 0, 'retries >= 0'))
+    volumes = field(type=PVector)
+
+    @property
+    def pod_name(self) -> str:
+        return self.name
```

Comment:
previously this was `task_id`, which consisted of `name` and `uuid`, that task_id was then sent to mesos to uniquely identify a task. in kubernetes the corresponding entity is a pod. you will probably want to keep the same approach with kubernetes pods, having `pod_name` consist of a meaningful part and a unique id because pod_name must be unique in a namespace.

strictly speaking, `uuid` attribute doesn't need to be of UUID type, it's just that in mesos, but any random sequence of ascii will do if we want something more compact.

Reply:
Done.